README.md

```diff
@@ -488,6 +488,7 @@ API | Description | Auth | HTTPS | CORS |
 | [PlayerUnknown's Battlegrounds](https://pubgtracker.com/site-api) | PUBG Stats | `apiKey` | Yes | Unknown |
 | [Pokéapi](https://pokeapi.co) | Pokémon Information | No | Yes | Unknown |
 | [Pokémon TCG](https://pokemontcg.io) | Pokémon TCG Information | No | Yes | Unknown |
+| [Pyschonauts API](https://pyschonauts-api.netlify.app/) | Pyschonauts World Characters Information and PSI Powers | No | No | Yes |
```

Comment:
```suggestion
| [Pyschonauts](https://pyschonauts-api.netlify.app/) | Pyschonauts World Characters Information and PSI Powers | No | Yes | Yes |
```

Reply:
No need to add API in the name. Ref. [contributing guide](https://github.com/public-apis/public-apis/blob/master/CONTRIBUTING.md)

Reply:
Noted, thanks for the feedback!

Reply:
Alright, done! I squashed my commits. But some new ones also popped up that weren't mine, if you check there's a commit that says: "Add URLHaus API to Anti-Maleware"., by FH089.

Its my first time squashing.